backend/lib/db.js

```diff
@@ -103,22 +83,10 @@ class DataBase {
     }
 
     findAll(collection, query) {
-        this.db.collection(collection).find(query, this.fieldsWanted).sort({date: -1}).toArray((err, result) => {
-            if (err) {
-                console.log('err', err)
-                return reject(err)
-            }
-            return resolve(result)
-        })
+        return this.db.collection(collection).find(query, this.fieldsWanted).sort({date: -1}).toArray()
     }
     findOne(collection, query) {
-        this.db.collection(collection).find(query).toArray((err, result) => {
-            if (err) {
-                console.log('err', err)
-                return reject(err)
-            }
-            return resolve(result[0])
-        })
+        return this.db.collection(collection).find(query).toArray()
```

Comment:
Same, can be replaced with: `this.db.collection(collection).findOne` instead of `find`

Reply:
yes saw that after i pushed made another push